dlc-manager/src/sub_channel_manager.rs

```diff
@@ -2026,7 +2039,8 @@ where
                 )?;
 
                 let sign_channel = dlc_messages::channel::SignChannel {
-                    channel_id: sub_channel_confirm.channel_id,
+                    // NOTE: I think this was wrong before!
```

Comment:
I think you're right. It didn't show up because this field is not used in `verify_signed_contract_internal` where this is passed.